client/src/components/SummaryCalculation.test.ts

```diff
@@ -0,0 +1,36 @@
+import { SAMPLE_ADDRESS_RECORDS } from "state-machine-sample-data";
+import { calculateAggDataFromAddressList, extractLocationDataFromAddr } from "./SummaryCalculation";
+
+describe("extractLocationDataFromAddr()", () => {
+  it("works", () => {
```

Comment:
lol mayyyybe a little more specificity here would be cool - like `it('gets all 5 fields from address')` or something